### PR TITLE
Added comprehension questions & feedback.md

### PR DESCRIPTION
**Pull request by CheezItMan**
Added comprehension questions and `feedback.md` for Startrly.  Let me know any suggested changes.

### CODE REVIEW

File: feedback.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+## What We're Looking For
+
+Feature | Feedback
+--- | ---
+**General** |
+Small commits with meaningful commit messages |
+Answered comprehension questions |
+**Code Requirements** |
+Only limited changes to the HTML |
+Page normalized |
```
Comment (droberts-sea) [inline feedback.md]: Changes should be applied to CSS, not HTML.

File: feedback.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+CSS Grid Used for layout |
+Flexbox used for layout |
+Google Font used |
```
Comment (droberts-sea) [inline feedback.md]: Can you use consistent capitalization through here?

File: feedback.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+CSS pseudo selector used |
+Background color |
+DRY CSS Code |
+Attractive page layout and appearance |
+**Overall** |
```
Comment (droberts-sea) [inline feedback.md]: I don't really care about "attractive" - for this assignment it's more important that it matches the image.

**Review by droberts-sea — CHANGES_REQUESTED** [review]

File: feedback.md (hunk 1 above)
Comment (CheezItMan) [reply feedback.md]: They can add classes to the HTML

**Pull request merged**